src/Orchard.Web/Modules/Orchard.ContentTypes/Views/DisplayTemplates/EditTypeViewModel.cshtml

```diff
@@ -9,7 +9,7 @@
     <div class="properties">
         <h3>@Model.DisplayName</h3> @if (!string.IsNullOrWhiteSpace(stereotype)) { <text><span class="stereotype" title="Stereotype">- @stereotype</span></text>}
         @if (creatable) {
-        <p class="pageStatus">@Html.ActionLink(T("Create New {0}", Model.DisplayName).Text, "Create", new {area = "Contents", id = Model.Name})</p>
+        <p class="pageStatus">@Html.ActionLink(T("Create New {0}", new HtmlString(Model.DisplayName)).Text, "Create", new {area = "Contents", id = Model.Name})</p>

```

Comment:
If I create the link with your method, it works because browser handles not encoded special characters, but the result is inconsistent.

French translation for "Create New {0}" is "Créer {0}"
My content name is "Représentation"
Generated markup is ``<a href="/Admin/Contents/Create/Representation">Créer Repr&#233;sentation</a>`` when it should be ``<a href="/Admin/Contents/Create/Representation">Cr&#233;er Repr&#233;sentation</a>``.

If we choose to encode translation in po file, it will not work for older versions.

I would want to understand the way we handle translations encoding, I opened a discussion [here](http://orchard.codeplex.com/discussions/642369) to talk about this.